Solutions/Training/Azure-Sentinel-Training-Lab/Modules/Module-1-Setting-up-the-environment.md

```diff
@@ -5,48 +5,73 @@
 
 ## Objectives
 
-This module guides you through the deployment of the Azure Sentinel environment that will be used in all subsequent modules.
+This module guides you through the deployment of the Azure Sentinel Training Lab solution that will be used in all subsequent modules.
 
 #### Prerequisites
 
-To get started with Azure Sentinel, you must have a Microsoft Azure subscription. If you do not have a subscription, you can sign up for a free account.
+To get started with Azure Sentinel, you must have a Microsoft Azure subscription. If you do not have a subscription, you can sign up for a free account [here](https://azure.microsoft.com/en-us/free/).
```

Comment:
Please update this link https://azure.microsoft.com/en-us/free/  to https://azure.microsoft.com/en/free. Website should not contain locale in it. Because of that checks are failing.

Reply:
Done @NikTripathi ! Thanks for noticing